internal/cmd/db_show.go

```diff
@@ -14,6 +14,7 @@ import (
 func init() {
 	dbCmd.AddCommand(showCmd)
 	showCmd.Flags().BoolVar(&showUrlFlag, "url", false, "Show URL for the database HTTP API.")
+	showCmd.Flags().BoolVar(&showHttpUrlFlag, "http-url", false, "Show HTTP URL for the database HTTP API.")
```

Comment:
I am not sure why is this needed?

Reply:
showHttpUrlFlag is defined bot not be register as flag in the cobra. 
https://github.com/tursodatabase/turso-cli/blob/5eeb66c637641426c94a6ee8659bc22939e9c508/internal/cmd/db.go#L17